internal/contributebot/webhook/README.md

```diff
@@ -0,0 +1,24 @@
+# Contribute Bot Webhook Listener
+
+Contribute Bot Webhook Listener is a microservice that listens for
+[GitHub webhook][] events and publishes them to a [Cloud Pub/Sub][] topic.
+
+[GitHub webhook]: https://developer.github.com/webhooks/
+[Cloud Pub/Sub]: https://cloud.google.com/pubsub/docs/publisher
+
+## Running Locally
```

Comment:
Mention GCP & App Engine SDK is required.

Reply:
Done.